client/i18n/locales/english/translations.json

```diff
@@ -347,7 +346,42 @@
     "try-again": "Please try again.",
     "card-number": "Your Card Number:",
     "expiration": "Expiration Date:",
-    "only-you": "Only you can see this message. Congratulations on earning this certification. It’s no easy task. Running freeCodeCamp isn’t easy either. Nor is it cheap. Help us help you and many other people around the world. Make a tax-deductible supporting donation to our nonprofit today."
+    "faq": "Frequently asked questions",
+    "only-you": "Only you can see this message. Congratulations on earning this certification. It's no easy task. Running freeCodeCamp isn't easy either. Nor is it cheap. Help us help you and many other people around the world. Make a tax-deductible supporting donation to our nonprofit today.",
+    "get-help": "How can I get help with my donations?",
+    "how-transparent": "How transparent is freeCodeCamp.org?",
+    "very-transparent": "Very. We have a Platinum transparency rating from GuideStar.org.",
+    "download-irs": "You can <0>download our IRS Determination Letter here</0>.",
+    "download-990": "You can <0>download our most recent 990 (annual tax report) here</0>.",
+    "how-efficient": "How efficient is freeCodeCamp?",
+    "fcc-budget": "freeCodeCamp's budget is much smaller than most comparable nonprofits. We haven't brought in professional fundraisers. Instead, Quincy does everything himself.",
+    "help-millions": "However, on a budget of only a few hundred thousand dollars per year, we have been able to help millions of people.",
+    "how-one-time":"How can I make a one-time donation?",
+    "one-time": "If you'd prefer to make one-time donations, you can support freeCodeCamp's mission whenever you have cash to spare. You can use <0>this link to donate whatever amount feels right through PayPal<0>.",
+    "wire-transfer": "You can also send money to freeCodeCamp directly through a wire transfer. If you need our wire details, email Quincy at quincy@freecodecamp.org",
+    "does-crypto": "Does freeCodeCamp accept donations in Bitcoin or other cryptocurrencies?",
+    "yes-crypto": "Yes, and we would welcome your cryptocurrency donations. Here are our wallet details:",
+    "can-check": "Can I mail a physical check?",
+    "yes-check": "Yes, we would welcome a check. You can mail it to us at:",
```

Comment:
```suggestion
    "can-cheque": "Can I mail a physical cheque?",
    "yes-cheque": "Yes, we would welcome a cheque. You can mail it to us at:",
```

Reply:
Sorry to say... But our preference for American spellings means this should be `check` 🙂

Reply:
Wow... I thought that was just plain incorrect